src/pattern-library/components/patterns/UtilityFoundations.js

```diff
@@ -0,0 +1,26 @@
+import Library from '../Library';
```

Comment:
Please don't worry too much about where this pattern-library page shows up or its nuances; I envision a re-org of pattern-library pages in the next sizeable chunk of work in this package.